os/drivers/audio/alc5658.c

```diff
@@ -672,10 +658,6 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct
 	case AUDIO_TYPE_PROCESSING:
 		break;
 	}
-
-	alc5658_setregs(priv);
-	alc5658_getregs(priv);
```

Comment:
I think getregs is useful for debuging, but definetly rename needed.

Reply:
Okay, will keep it.